### ISSUE TITLE
Desktop app can't connect to node while mobile can

### ISSUE THREAD

**Issue by chompomonim** [mysteriumnetwork/node#2202]
There are nodes to which it is possible connect with Android app, while from same network it's not possible to connect from desktop app.

e.g. node: 0xe84eb3c9d51f289ac939c7a97dc40003007ea077

**Environment (please complete the following information):**
 - Node version: 0.31.3 on RPi
 - Android client: 0.31.3
 - Desktop app: 2.0.0-beta1

**Desktop app logs**
```
2020-05-12T10:06:58.778 INF Users/tadas/mygo/node/core/connection/manager.go:624 > Connection state: NotConnected -> Connecting
2020-05-12T10:06:58.778 DBG Users/tadas/mygo/node/communication/nats/connector.go:45 > Connecting to NATS servers: nats://testnet-broker.mysterium.network:4222
2020-05-12T10:06:58.779 DBG Users/tadas/mygo/node/eventbus/event_bus.go:81 > Published topic="State" event={State:Connecting SessionInfo:{StartedAt:2020-05-12 10:06:58.77873 +0300 EEST m=+39051.734183778 ConsumerID:{Address:0x5c8cbe8f1c64ebe51df4e5bed690d41e8c1bb0fb} AccountantID:[2 20 40 28 241 92 26 102 181 25 144 226 230 94 31 123 124 54 51 24] State:Connecting SessionID: Proposal:{ID:1 Format:service-proposal/v1 ServiceType:openvpn ServiceDefinition:{Location:{Continent:EU Country:PL City:Greater Poland ASN:197227 ISP:Poznanska Spoldzielnia Mieszkaniowa Winogrady w Po NodeType:residential} LocationOriginate:{Continent:EU Country:PL City:Greater Poland ASN:197227 ISP:Poznanska Spoldzielnia Mieszkaniowa Winogrady w Po NodeType:residential} SessionBandwidth:8.388608e+07 Protocol:udp} PaymentMethodType:BYTES_TRANSFERRED_WITH_TIME PaymentMethod:{Price:0.000500MYST Duration:1m0s Bytes:7669584 Type:BYTES_TRANSFERRED_WITH_TIME} ProviderID:0xe84eb3c9d51f289ac939c7a97dc40003007ea077 ProviderContacts:[{Type:nats/v1 Definition:{Topic:0xe84eb3c9d51f289ac939c7a97dc40003007ea077.openvpn BrokerAddresses:[nats://testnet-broker.mysterium.network:4222]}} {Type:nats/p2p/v1 Definition:{BrokerAddresses:[nats://testnet-broker.mysterium.network:4222]}}] AccessPolicies:<nil>}}}
2020-05-12T10:06:58.779 INF Users/tadas/mygo/node/firewall/outgoing_firewall_noop.go:57 > Allow URL nats://testnet-broker.mysterium.network:4222 access
2020-05-12T10:06:58.779 INF Users/tadas/mygo/node/core/state/state.go:359 > Session ID  Connecting duration: 549.053µs data: 0b/0b, throughput: 0bs/0bs, spent: 0.000000MYST
2020-05-12T10:06:58.888 DBG Users/tadas/mygo/node/p2p/dialer.go:144  > Consumer 0x5c8cbe8f1c64ebe51df4e5bed690d41e8c1bb0fb sending public key e3d4d3e3e57cabdc4adad0d402e13b4b1cbdc16cb9f31b76957a72fc1349b763 to provider 0xe84eb3c9d51f289ac939c7a97dc40003007ea077
2020-05-12T10:07:00.519 DBG Users/tadas/mygo/node/p2p/dialer.go:171  > Consumer 0x5c8cbe8f1c64ebe51df4e5bed690d41e8c1bb0fb received provider 0xe84eb3c9d51f289ac939c7a97dc40003007ea077 with config: publicIP:"213.5.214.134"  ports:44057  ports:46647
2020-05-12T10:07:00.820 DBG Users/tadas/mygo/node/core/ip/resolver.go:103 > IP detected: 78.61.246.140
2020-05-12T10:07:00.820 INF Users/tadas/mygo/node/core/port/pool.go:68 > Supplying port 45894
2020-05-12T10:07:00.820 INF Users/tadas/mygo/node/core/port/pool.go:68 > Supplying port 40545
2020-05-12T10:07:00.821 DBG Users/tadas/mygo/node/p2p/dialer.go:194  > Consumer 0x5c8cbe8f1c64ebe51df4e5bed690d41e8c1bb0fb sending ack with encrypted config to provider 0xe84eb3c9d51f289ac939c7a97dc40003007ea077
2020-05-12T10:07:00.984 INF Users/tadas/mygo/node/firewall/outgoing_firewall_noop.go:48 > Allow IP 213.5.214.134 access
2020-05-12T10:07:00.984 DBG Users/tadas/mygo/node/p2p/dialer.go:99   > Skipping provider ping
2020-05-12T10:07:00.985 DBG Users/tadas/mygo/node/p2p/dialer.go:121  > Received handlers ready message from provider
2020-05-12T10:07:00.985 WRN Users/tadas/mygo/node/communication/nats/connection_wrap.go:86 > NATS: disconnected
2020-05-12T10:07:00.985 INF Users/tadas/mygo/node/communication/nats/connector.go:71 > Removing broker connection from the registry: c2b6f937-054d-4e57-9643-3a2a3d385703
2020-05-12T10:07:00.985 WRN Users/tadas/mygo/node/communication/nats/connection_wrap.go:85 > NATS: connection closed
2020-05-12T10:07:00.986 INF Users/tadas/mygo/node/firewall/outgoing_firewall_noop.go:61 > Rule for URL: nats://testnet-broker.mysterium.network:4222 removed
2020-05-12T10:07:00.986 ERR Users/tadas/mygo/node/core/connection/manager.go:191 > Connect failed, disconnecting error="could not create p2p channel: could not create p2p channel: could not reopen remote conn: could not listen UDP: listen udp4 192.168.0.134:45894: bind: can't assign requested address"
2020-05-12T10:07:00.986 INF Users/tadas/mygo/node/core/connection/manager.go:624 > Connection state: Connecting -> NotConnected
2020-05-12T10:07:00.986 ERR Users/tadas/mygo/node/tequilapi/endpoints/connection.go:185 >  error="could not create p2p channel: could not create p2p channel: could not reopen remote conn: could not listen UDP: listen udp4 192.168.0.134:45894: bind: can't assign requested address"
2020-05-12T10:07:00.986 DBG Users/tadas/mygo/node/eventbus/event_bus.go:81 > Published topic="State" event={State:NotConnected SessionInfo:{StartedAt:2020-05-12 10:06:58.77873 +0300 EEST m=+39051.734183778 ConsumerID:{Address:0x5c8cbe8f1c64ebe51df4e5bed690d41e8c1bb0fb} AccountantID:[2 20 40 28 241 92 26 102 181 25 144 226 230 94 31 123 124 54 51 24] State:NotConnected SessionID: Proposal:{ID:1 Format:service-proposal/v1 ServiceType:openvpn ServiceDefinition:{Location:{Continent:EU Country:PL City:Greater Poland ASN:197227 ISP:Poznanska Spoldzielnia Mieszkaniowa Winogrady w Po NodeType:residential} LocationOriginate:{Continent:EU Country:PL City:Greater Poland ASN:197227 ISP:Poznanska Spoldzielnia Mieszkaniowa Winogrady w Po NodeType:residential} SessionBandwidth:8.388608e+07 Protocol:udp} PaymentMethodType:BYTES_TRANSFERRED_WITH_TIME PaymentMethod:{Price:0.000500MYST Duration:1m0s Bytes:7669584 Type:BYTES_TRANSFERRED_WITH_TIME} ProviderID:0xe84eb3c9d51f289ac939c7a97dc40003007ea077 ProviderContacts:[{Type:nats/v1 Definition:{Topic:0xe84eb3c9d51f289ac939c7a97dc40003007ea077.openvpn BrokerAddresses:[nats://testnet-broker.mysterium.network:4222]}} {Type:nats/p2p/v1 Definition:{BrokerAddresses:[nats://testnet-broker.mysterium.network:4222]}}] AccessPolicies:<nil>}}}
2020-05-12T10:07:00.986 DBG Users/tadas/mygo/node/core/location/oracle_resolver.go:41 > Detecting with oracle resolver
2020-05-12T10:07:00.986 INF Users/tadas/mygo/node/core/state/state.go:359 > Session ID  NotConnected duration: 2.208193484s data: 0b/0b, throughput: 0bs/0bs, spent: 0.000000MYST
2020-05-12T10:07:01.320 DBG Users/tadas/mygo/node/core/location/cache.go:95 > Location update succeeded: {78.61.246.140 8764 Telia Lietuva, AB EU LT Vilnius residential}
2020-05-12T10:07:01.320 DBG Users/tadas/mygo/node/core/location/oracle_resolver.go:41 > Detecting with oracle resolver
```

**Comment by soffokl** [comment]
I did a try to connect from CLI, both OpenVPN and Wireguard seem too be working fine.
![image](https://user-images.githubusercontent.com/8612618/81652169-24dfd000-9455-11ea-9bc3-df3947c3c552.png)

**Comment by chompomonim** [comment]
@soffokl could you try fro desktop app? It is still could be related either to mine network or to desktop app itself.

**Comment by anjmao** [comment]
I will look into it.

**Comment by zolia** [comment]
Looks like your private IP does not exists at the time of session setup. Please send us your routing and interface tables at the moment session is being setup.
On MacOS: 
`# netstat -r`
`# ifconfig`

**Comment by chompomonim** [comment]
Closing as it should be fixed with #2233
